ebpf-io/ebpf.io-website
# feat: added french version of pages

**Describe what changes this pull request brings**
This pull request adds the French version of the pages.

**Steps to test**
1. Open the [Labs Page](https://deploy-preview-482--ebpfio.netlify.app/fr-fr/labs/), [Applications Page](https://deploy-preview-482--ebpfio.netlify.app/fr-fr/applications/), [Infrastructure Page](https://deploy-preview-482--ebpfio.netlify.app/fr-fr/infrastructure/)
1. Confirm that everything looks and works as expected
